build/utils/npm.go

```diff
@@ -410,9 +410,13 @@ func GetNpmVersionAndExecPath(log utils.Log) (*version.Version, string, error) {
 }
 
 type PackageInfo struct {
-	Name    string `json:"name,omitempty"`
-	Version string `json:"version,omitempty"`
-	Scope   string
+	Name                 string            `json:"name,omitempty"`
+	Version              string            `json:"version,omitempty"`
+	Dependencies         map[string]string `json:"dependencies"`
+	DevDependencies      map[string]string `json:"devDependencies"`
+	PeerDependencies     map[string]string `json:"peerDependencies"`
+	OptionalDependencies map[string]string `json:"optionalDependencies"`
+	Scope                string
```

Comment:
why no json field?

Reply:
there is no Scope field in package.json files therefore the Scope field is not intended for the json parsing